src/screens/Signup/styles.module.scss

```diff
@@ -0,0 +1,31 @@
+@import "scss/variables/_colors";
+
+.container {
+  background-color: $wild-sand;
+  border-top: 8px solid $wolox-blue;
+  margin-top: 51px;
+  max-width: 300px;
+  padding: 20px 0 24px;
+
+  img {
+    width: 212px;
+    margin-bottom: 16px;
+  }
+
+  .container-form {
+    padding: 0 23px 0 25px;
+
+    .form {
+      border-bottom: 2px solid $silver-chalice;
+      margin-bottom: 16px;
+      padding-bottom: 16px;
+
+      .field {
```

Comment:
field should be `label`, also `label` is a part of the reusable `Input` component. Remember also the important of accessibility, and if you are using a text for describing an input, it should be a label containing `labelFor` and pass the name and also use the id for make the click the label focus the input